yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClientSchedulerBackend.scala

```diff
@@ -125,6 +125,7 @@ private[spark] class YarnClientSchedulerBackend(
    */
   private def asyncMonitorApplication(): Unit = {
     assert(client != null && appId != null, "Application has not been submitted yet!")
+    val interval = conf.getLong("spark.yarn.client.progress.pollinterval", 1000)
```

Comment:
Yeah, you are right. In PR #5305 I use the client.monitorApplication, then we can use "spark.yarn.report.interval" to changing the yarn client monitor interval. Thank you.